src/core_atmosphere/physics/mpas_atmphys_driver_lsm.F

```diff
@@ -725,6 +735,9 @@ subroutine lsm_to_MPAS(configs,mesh,diag_physics,sfc_input,its,ite)
    call mpas_pool_get_array(diag_physics,'smfr3d'         ,smfr3d     )
    call mpas_pool_get_array(diag_physics,'keepfr3d'       ,keepfr3d   )
    call mpas_pool_get_array(diag_physics,'mavail'         ,mavail     )
+   call mpas_pool_get_array(diag_physics,'t2m'            ,t2m        )
+   call mpas_pool_get_array(diag_physics,'th2m'           ,th2m       )
+   call mpas_pool_get_array(diag_physics,'q2'             ,q2         )
```

Comment:
Mike, t2, th2m and q2 are the left overs since the time when 2-m diagnostics was called from the LSM driver. They should be completely removed from the LSM driver.

Reply:
@tanyasmirnova I maintained your code as much as possible, just isolated your changes to these ruc blocks. But I'm all for removing as much unnecessary code as possible. I will purge these from the lsm driver.